uwuno/src/main/java/com/learning/uwuno/game/gameState.java

```diff
@@ -0,0 +1,151 @@
+package com.learning.uwuno.game;
+
+import com.learning.uwuno.cards.basicCard;
+import com.learning.uwuno.cards.card;
+import com.learning.uwuno.cards.wildCard;
+import com.learning.uwuno.player;
+import com.learning.uwuno.room;
+import com.learning.uwuno.util.playerList;
+
+import java.util.*;
+
+public class gameState {
+    private int cardsToDraw;    // Keeps track of +4, +2 stacking
+    private int turnsTaken;
+    private boolean turnDirection; // true if forward, otherwise reverse direction
```

Comment:
Use an enum for this.

Reply:
Is there a reason to make them enums? Shouldn't enums be final/static?

Reply:
Just that it would be easier to read the code in the future, since eventually we're gunna just see a true or false while debugging and it'll confuse us. Enums don't have to be static, or at least in some of the code we already have, non static enums.

Reply:
In that case, I'll just rename the variable to something more intuitive.